src/test/resources/application-default.yml

```diff
@@ -1,3 +1,19 @@
+spring:
+  jpa:
+    database: H2
+    hibernate:
+      ddl-auto: create-drop
+    properties:
+      dialect: org.hibernate.dialect.H2Dialect
+      hibernate.format_sql: true
+
+logging:
+  level:
+    org.springframework.orm.jpa: DEBUG
+    org.springframework.transaction: DEBUG
+    org.hibernate.SQL: DEBUG
+    org.hibernate.type.descriptor.sql.BasicBinder: TRACE
+
 dbVendor: h2:mem
 query: ;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE
 hostPort: ":///"
```

Comment:
hostPort: ":"   like in another postgres services (study-server, network-modification-server ...) ?

Reply:
done